Clean merged exp (from PR #830)

jm-c opened the pull request:
## What changes does this PR introduce?
Remove/cleaning remaining files and input dir after few exp. got merged in PR #830, specifically:
1. FWD tests:

-  flt_example                   $\rightarrow$  exp4.with_flt
- global_with_exf             $\rightarrow$  global_ocean_ebm.w_exf
- global_with_exf.yearly $\rightarrow$  global_ocean_ebm.yearly
- natl_box                         $\rightarrow$  lab_sea.natl_box
- natl_box.longstep        $\rightarrow$  lab_sea.longstep

2. TAF & Tapenade tests:
- global_with_exf            $\rightarrow$  global_oce_latlon.w_exf

## What is the current behaviour? 
merged exp (flt_example, global_with_exf, natl_box) were left in place with code, input, and renamed ref. results output.

## What is the new behaviour 
finish merging: remove un-used and untested code, move input dir and ref. output (replacing symbolic links).

## Does this PR introduce a breaking change? 
no

## Other information:

## Suggested addition to `tag-index`
o verification:
  - After merging (5 fwd tests + 2 AD) few exp. into new host-exp. (PR #830),
    move input dir and ref. output from merged exp. to replace sym-link in new
    host exp. and remove remaining files/dir (code, build, run) in merged-exp;
  - Add or update README.md in new host-exp and update list of exp. in the docs.

jm-c commented:
Next step: update some "README(.md)" and update list of exp. in the docs.

jm-c commented:
I am done with the README and docs updates and this is ready for review.

jrscott commented:
@jm-c  I think stuffing the exf experiments into global_ocean_ebm without changing the name is not best; the most useful global verif setups are now hidden under the name of something little used.
Also, do we need to keep the confusing former name references and directory links this way?

Can we maybe change the name of global_ocean_ebm to global_ocean_forcpkg to more accurately represent the overall suite of experiments? Or suggest something not *_ebm ?

jm-c commented:
@jrscott regarding the addition of this reference/link for the APE (secondary test) experiment `atm_gray.ape`: Since "MIT GCM" is listed as a "participant" there, but these results correspond to a different  experiment, in this case `fizhi-cs-aqualev20/`, it might introduce some confusion. If we want to add a reference for APE for `atm_gray.ape` test experiment, may be we could just mention the  Neale and Hoskins 2001 paper, doi:10.1006/asle.2000.0019. And the reference you added could be moved to  `fizhi-cs-aqualev20/` description ?

jrscott commented:
I'll swap in the reference rather than the URL. This project seems 10+ years dead at this point anyway.

jm-c commented:
@jahn You could check that this PR indeed remove all the symbolic-links that you found confusing.
@mjlosch I assume you don't have much time now to look at this in details, but in case.
This PR is just cleaning a little bit the set of verification experiments (zero code change) and has been waiting since Nov 14 (ready and documented).

jahn reviewed (approved):
It does beautifully clean up the links!

jm-c commented:
I am planing to merge this PR soon, like tomorrow, unless someone wants to add something.

jrscott commented:
@jm-c can you hold off merging at least until an issue is created

jm-c commented:
@jrscott  I have been waiting for this PR to be merged since Nov 14, and Oliver approved it on Feb 27. No changes to the code,
no changes to testreport results, just cleaning.

jm-c commented:
@jrscott few more comments:

I think this PR follows the broadly accepted guideline, and as mentioned earlier, I was hoping to get this merged soon (this morning):
1. the main purpose got announced on May 03, 2024, as a first comment to PR #830 :
> 2. my intention was, for this PR, to leave the input and code dir in the 3 original experiments, with the symbolic links in the new host experiments. And would open a new PR just to clean-up and finish the move. This way the old set-up is still accessible to compare with the new set of tests.
> 3. regarding documentation update, could wait for this future/new PR since the old/original set-up will still be available until then (and the new "input" dir is just a link).

And indeed this PR fulfill these 2 objectives, nothing less, nothing more.

Furthermore, few "README.md" were added or converted (from plain text README) or updated, thus improving some verification experiment documentation.

It was ready for review on Nov 14, 2024, when I was done updating the README, and removed the "work-in-progress" label.

2. the review process did follow the normal path. You made some changes to the docs with few further adjustments. And I did respond in details (285 words) on Feb 25 to your short comments (72 words) you wrote on Feb 19: I addressed some of your concern and proposed some suggestions to specific points.

    So after @jahn approval (on Feb 27), and after waiting a couple of weeks for any reply from my Feb 19 response, I thought it was time to merge this cleaning PR (as mentioned earlier, no code changes, no changes to testreport output) and so, yesterday, posted that I would merge it today.

Now, as I was about to merge this PR (already updated tag-index) it seems that you have new comments (a request for a new review) to add here. I hope to see these new comments soon, so that we can be done with this PR by the end of this week (before St Patrick), and move forward.

jrscott commented:
offline @jm-c and I realized the softlink confusion was that it requires the PR merged to master before it will "work" in the doc -- ie issue resolved

will keep the ref link to old exp name in doc in perpetuity, given that the old names have existed a very long time

see new issue #907 regarding the exf experiments under global_ocean_ebm  umbrella 

LGTM

jm-c commented:
@jrscott Thanks. Will merge this PR soon.
